roottest/root/dataframe/CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ endif()
 
 set(DFLIBRARIES Core RIO Hist Tree MathCore TreePlayer ROOTDataFrame ROOTVecOps)
 
-if(ROOT_root7_FOUND)
+if(root7)
   list(APPEND DFLIBRARIES ROOTNTuple)
 endif()
```

Comment:
Nowadays this `if` is not needed anymore, `ROOTNTuple` is built by default, can you remove the if?